### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dwu359**
https://github.com/DSGT-DLP/Deep-Learning-Playground/pull/990

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-## Windows installation
+## Windows installation (Do this if WSL doesn't work)
+
+Step 1: If you don't have the `dlp-cli` submodule, run `git submodule init dlp-cli` in the project directory. Then run `git submodule update`. If it says that you are in a detached HEAD state, cd to the dlp-cli submodule directory and `git checkout main`.
 
-Step 1: If you don't have the `dlp-cli` submodule, run `git submodule init dlp-cli` in the project directory. Then run `git submodule update`.
+Step 2: Run `winget settings --enable LocalManifestFiles` as administrator. Then, in the project directory, run `winget install -m dlp-cli/manifests/d/DSGT-DLP/dlp-cli/{version}`. Replace `{version}` with the version number that you want (typically the latest version).
 
-Step 2: In the project directory, run `winget install -m dlp-cli/manifests/d/DSGT-DLP/dlp-cli/{version}`. Replace `{version}` with the version number that you want (typically the latest version).
+Step 3: Add the path to the install location to your PATH environment variable (somewhere within `%localappdata%\microsoft\winget\packages`, may need file explorer to find the exact folder)
```
Comment (karkir0003) [inline README.md:21]: i think the %localappdata thingy is part of the path variable. pelase make sure the casing is correct bc path variables are usually case sensitive @dwu359 @noah-iversen

**Review by karkir0003 — CHANGES_REQUESTED** [review]
address my comment

File: README.md (hunk 1 above)
Comment (DSGT-DLPLAYGROUND) [reply README.md:21]: Noah said that it was correct

**Review by farisdurrani — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (karkir0003) [reply README.md:21]: nice

**Pull request merged**